libr/core/cmd_debug.inc.c

```diff
@@ -4926,8 +4926,17 @@ static int cmd_debug_continue(RCore *core, const char *input) {
 		}
 		break;
 	case ' ':
+		const char *pidstr = r_str_trim_head_ro (input + 2);
+		if (!pidstr || !*pidstr) {
```

Comment:
you can't define vars after ':' in the case. open braces, move that into a separtte function or use ":;"